app/tests/spec/models/auth_brokers/fx-desktop-v4.js

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define((require, exports, module) => {
+  'use strict';
+
+  const { assert } = require('chai');
+  const FxDesktopV4AuthenticationBroker = require('models/auth_brokers/fx-desktop-v4');
+  const p = require('lib/promise');
+  const sinon = require('sinon');
+  const WindowMock = require('../../../mocks/window');
+
+  describe('models/auth_brokers/fx-desktop-v4', () => {
+    let broker;
+    let windowMock;
+
+    beforeEach(() => {
+      windowMock = new WindowMock();
+
+      broker = new FxDesktopV4AuthenticationBroker({
+        window: windowMock
+      });
+    });
+
+    describe('capabilities', () => {
+      it('has the `allowUidChange` capability', () => {
```

Comment:
This test title is the only occurrence of "allowUidChange" in the PR, is it a hangover from a previous naming convention?